### PR TITLE
[Release] Release of vizro-ai 0.2.1

### PR DESCRIPTION
**Pull request by lingyielia**
## Notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

  - I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
  - I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorized to submit this contribution on behalf of the original creator(s) or their licensees.
  - I certify that the use of this contribution as authorized by the Apache 2.0 license does not violate the intellectual property rights of anyone else.
  - I have not referenced individuals, products or companies in any commits, directly or indirectly.
  - I have not added data or restricted code in any commits, directly or indirectly.

### CODE REVIEW

File: vizro-ai/CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,28 @@ See the fragment files in the [changelog.d directory](https://github.com/mckinse
...
+
+# 0.2.1 — 2024-08-28
+
+## Removed
+
+- Private attribute `_return_all_text` is removed from `VizroAI` class ([#518](https://github.com/mckinsey/vizro/pull/518))
+
+## Added
+
+- Enable feature to get PlotOutputs dataclass from `VizroAI.plot()`. Add argument `return_elements`to`VizroAI.plot()`, when it is set to`True`, the return type will be changed to a `dataclass` containing the code string, figure object, business insights, and code explanation. ([#488](https://github.com/mckinsey/vizro/pull/488))
```
Comment (maxschulz-COL) [inline vizro-ai/CHANGELOG.md:24]:
```suggestion
- Add argument `return_elements` to`VizroAI.plot()`. When it is set to `True`, the return type will be changed to a `dataclass` containing the code string, figure object, business insights, and code explanation. ([#488](https://github.com/mckinsey/vizro/pull/488))
```

File: vizro-ai/CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,28 @@ See the fragment files in the [changelog.d directory](https://github.com/mckinse
...
+
+- Add Google Colab notebook for users to experiment with `VizroAI.plot` method without local setup ([#618](https://github.com/mckinsey/vizro/pull/618))
+
```
Comment (maxschulz-COL) [inline vizro-ai/CHANGELOG.md:27]:
```suggestion
```

File: vizro-ai/CHANGELOG.md [hunk 3]
```diff
@@ -11,6 +11,28 @@ See the fragment files in the [changelog.d directory](https://github.com/mckinse
...
+- Added functionality to generate dashboards from text. This feature is currently in **Alpha** and is not yet officially released ([#651](https://github.com/mckinsey/vizro/pull/651))
```
Comment (maxschulz-COL) [inline vizro-ai/CHANGELOG.md:28]:
```suggestion
- Add functionality to generate dashboards from text. This feature is currently in **Alpha** and is not yet officially released ([#651](https://github.com/mckinsey/vizro/pull/651))
```

File: vizro-ai/CHANGELOG.md [hunk 4]
```diff
@@ -11,6 +11,28 @@ See the fragment files in the [changelog.d directory](https://github.com/mckinse
...
+
+## Changed
+
+- Disabled figure display upon variable assignment, to display outcome of `VizroAI.plot()` add `.show()`. Enable feature XXX ([#527](https://github.com/mckinsey/vizro/pull/527))
```
Comment (maxschulz-COL) [inline vizro-ai/CHANGELOG.md:32]:
```suggestion
- Disabled figure display upon variable assignment. To display outcome of `VizroAI.plot()` add `.show()`. ([#527](https://github.com/mckinsey/vizro/pull/527))
```

**Review by maxschulz-COL — APPROVED** [review]
Approved pending the comments

**Review by nadijagraca — APPROVED** [review]

**Pull request merged**